main.py

```diff
@@ -142,7 +154,7 @@ async def unban(ctx, *, member):
             return
 
 
-# Commands which give information about certain topics and usable by all member is given below this line. #
```

Comment:
That was not a typo. I am wrapping text with two octothorps if they are header text.